Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3 as base
+
+#ENV LIBRARY_PATH=/lib:/usr/lib
+
+RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+
+WORKDIR /todo_app
+
+COPY . /todo_app
+#copy dir
+ENV PATH="${PATH}:/root/.poetry/bin"
+RUN poetry install
+
+#EXPOSE 80
+#install poetry
+FROM base as development 
+ENTRYPOINT  poetry run flask run --host 0.0.0.0
+
+FROM base as production
+CMD ["gunicorn"  , "-b", "0.0.0.0:8000", "app:app"]
```

Comment:
I see a few issues trying to run this, does it work for you?

* Gunicorn isn't found; we should Install gunicorn (e.g. with `poetry add gunicorn`) so that it's available to run
* Run it within your virtual environment, so using `poetry run gunicorn ...`
* Update the gunicorn command itself to use the right module; `app:app` implies your code is in a file called app.py within the same folder (but we've got that file inside a `todo_app` folder) which has a variable called `app` to run, but we want to use the `create_app` function, so we want to update it to be more like:
```
..., "todo_app.app:create_app()"]
```